.lintstagedrc

```diff
@@ -1,5 +1,6 @@
 {
   "*.js": [
+    "yarn run prettier --write",
```

Comment:
should this be done for ts files too?

Reply:
ah, I think this is already fixed in a later commit (I was reviewing it commit by commit)